### PR TITLE
Documentation corrections for LocalBondProjection and LocalDensity.

### LINKED ISSUE #335: LocalBondProjection documentation does not mention nlist
**Describe the bug**
The documentation for `LocalBondProjection` does not mention that one can access the computed neighbor list by the attribute `nlist`.

### PR DESCRIPTION
**Pull request by jinsooihm**
Resolves #335.
Also, fixed the image in `LocalDensity` to reflect what is being done.

### CODE REVIEW

File: doc/source/images/gen_density_graphic.py [hunk 1]
```diff
@@ -12,8 +12,9 @@
 ax.set_xticks(x[1:4])
 ax.set_yticks(y[1:4])
 fontdict = {'fontsize': 14}
-ax.set_xticklabels(["$r_{cut} - d$", "$r_{cut}$", "$r_{cut} + d$"], fontdict)
-ax.set_yticklabels(["$1.0$", "$0.5$", "$0$"], fontdict)
+ax.set_xticklabels([r"$r_{cut} - \frac{d}{2}$", r"$r_{cut}$",
+                    r"$r_{cut} + \frac{d}{2}$"], fontdict)
+ax.set_yticklabels(["$1.0$", "$0.5$", "$0$"])
```
Comment (bdice) [inline doc/source/images/gen_density_graphic.py]: Sorry if this is too pedantic... can you revert the change on this line, so that it says `fontdict` and uses the larger font? The y-axis ticks were more legible the other way. Then this is good to merge.

**Review by bdice — CHANGES_REQUESTED** [review]
One minor change requested.

File: doc/source/images/gen_density_graphic.py (hunk 1 above)
Comment (vyasr) [reply doc/source/images/gen_density_graphic.py]: @jinsoo960 let me know when this is fixed, it looks good otherwise so I can merge.

Comment (jinsooihm) [reply doc/source/images/gen_density_graphic.py]: It is changed.

**Review by bdice — APPROVED** [review]
Thanks @jinsoo960!
